### PR TITLE
HH-136: Carousel fixes

### PR DESCRIPTION
**Pull request by melniiv**
## Description
Carousel fixes needed for hobbies front page to work with new cards data

### Closes

**[DEV-XXX](https://helsinkisolutionoffice.atlassian.net/browse/DEV-XXX):**

### CODE REVIEW

File: src/core/collection/Collection.tsx [hunk 1]
```diff
@@ -159,14 +160,15 @@ function getEventCollectionCards(
...
-  const cards = getCollectionCards(generalCollection).map((cardProps) => {
+  const cards = getCollectionCards(generalCollection).map((cardProps, i) => {
     const url = getRoutedInternalHref(cardProps.url, null);
     return (
       <Card
         key={cardProps.id}
         {...cardProps}
         url={url}
         direction="fixed-vertical"
+        customContent={<EventCardContent event={items[i]} />}
```
Comment (nikomakela) [inline src/core/collection/Collection.tsx:171]: trivial: event/item could be added to a new variable

File: src/core/configProvider/configContext.ts [hunk 2]
```diff
@@ -31,6 +31,8 @@ export type Config = {
       props: React.AnchorHTMLAttributes<HTMLAnchorElement>,
     ) => JSX.Element;
     Head?: (props: { children: React.ReactNode }) => JSX.Element;
+    EventCardContent?: (props: any) => JSX.Element;
```
Comment (nikomakela) [inline src/core/configProvider/configContext.ts:34]: Unexpected any. Could it be `CardProps`? Is the real issue in rollup-package?

**Review by nikomakela — APPROVED** [review]
Trivial typing issues, which actually might be because of failing rollup-package, but changes are good!

File: src/core/configProvider/configContext.ts (hunk 2 above)
Comment (melniiv) [reply src/core/configProvider/configContext.ts:34]: no, actually it should be the event item...

File: src/core/collection/Collection.tsx (hunk 1 above)
Comment (melniiv) [reply src/core/collection/Collection.tsx:171]: i ll try to fix, maybe typing as well

**Pull request merged**